packages/website/plugins/generated-rule-docs.ts

```diff
@@ -40,22 +46,211 @@ const generatedRuleDocs: Plugin = () => {
+      );
+
+      if (optionsH2Index === -1) {
+        const whenNotToUseItH2Index = parent.children.findIndex(
+          createH2TextFilter('When Not To Use It'),
+        );
+        optionsH2Index =
+          whenNotToUseItH2Index === -1
+            ? parent.children.length - 1
+            : whenNotToUseItH2Index;
+
+        parent.children.splice(optionsH2Index, 0, {
+          children: [
+            {
+              type: 'text',
+              value: 'Options',
+            },
+          ],
+          depth: 2,
+          type: 'heading',
+        } as mdast.Heading);
+      }
+
+      if (meta.schema.length === 0) {
+        parent.children.splice(
+          optionsH2Index + 1,
+          0,
+          {
+            lang: 'jsonc',
+            type: 'code',
+            value: `// .eslintrc.json
+{
+  "rules": {
+    "@typescript-eslint/${file.stem}": "${optionLevel}"
+  }
+}`,
+          } as mdast.Code,
+          {
+            children: [
+              {
+                type: 'text',
+                value: 'This rule is not configurable.',
+              },
+            ],
+            type: 'paragraph',
+          } as mdast.Paragraph,
+        );
+      } else {
+        const optionsInterface = await compile(
+          {
+            title: `Options`,
+            ...(meta.schema[0] as JSONSchema4Object),
+          },
+          file.stem,
+          { additionalProperties: false, bannerComment: '' },
+        );
+
+        parent.children.splice(optionsH2Index + 1, 0, {
+          lang: 'ts',
+          type: 'code',
+          value: [
+            optionsInterface.replace(/^export /g, ''),
+            format(
+              `const defaultOptions: Options = ${JSON.stringify(
+                rule.defaultOptions,
+              )};`,
+              {
+                parser: tseslintParser.parse,
+              },
+            ),
+          ]
+            .join(EOL)
+            .trim(),
+        } as mdast.Code);
+      }
+    }
+
+    // 5. Add a notice about coming from ESLint core for extension rules
+    if (meta.docs.extendsBaseRule) {
+      parent.children.push(
+        {
+          type: 'html',
+          value: '<sup>',
+        } as mdast.HTML,
+        {
+          children: [
+            {
+              type: 'text',
+              value: 'Taken with ❤️ ',
+            },
+            {
+              children: [
+                {
+                  type: 'text',
+                  value: 'from ESLint core',
+                },
+              ],
+              type: 'link',
+              url: 'https://github.com/eslint/eslint/blob/main/docs/rules/require-await.md',
+            },
+          ],
+          type: 'paragraph',
+        } as mdast.Paragraph,
+        {
+          type: 'html',
+          value: '</sup>',
+        } as mdast.HTML,
+      );
+    }
+
+    // TODO: find & use library that converts text to markdown ast
+    // (that'll fix the <sup> not applying issue)
```

Comment:
Docusaurus uses MDX, so you need to use `type: "jsx"` instead. See the `<rule-attributes />` node anove. You can also experiment here: https://mdx-git-renovate-babel-monorepo-mdx.vercel.app/playground

Reply:
Aha, thanks!

Reply:
Err @Josh-Cena I tried changing to `{ type: 'jsx', value: '<sup>'/'</sup>' }` per the playground but it's not showing up. I also tried a `type: 'jsx'` node with a few variants of the raw HTML and/or MD(X), and no luck. What should this look like?

Reply:
I would take a look later 🤔 It should work like that

Reply:
For now, just to get this reviewable, I just made this a regular paragraph.